client/src/components/Reviews.jsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import './Reviews.css';
```

Comment:
@gorakong I noticed you are using CSS modules in your App component, but you are just importing the CSS and applying a class name here? Have you decided how you want to handle applying classes to your components?

Reply:
@jcelder  This will be fixed in my other commit under the 'styling' branch!